package.json

```diff
@@ -41,9 +41,21 @@
     "standard": "^16.0.3"
   },
   "devDependencies": {
+    "@types/jest": "^26.0.23",
     "@types/react": "^17.0.3",
+    "jest": "^27.0.3",
     "react": "^17.0.2",
     "ts-node": "^9.1.1",
     "typescript": "^4.1.2"
+  },
+  "jest": {
```

Comment:
**Nitpicking:**
Would it make sense to move this into a jest.config.json instead? Personally I don't like a bloated package.json